config/environments/production.rb

```diff
@@ -36,7 +36,8 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  config.active_storage.service = :cloudinary
```

Comment:
Important if you want to use Cloudinary in production and not only in your local machine